### PR TITLE
r? ianbicking Render react head and body, not document

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: server/src/views/frame.js [hunk 1]
```diff
@@ -430,16 +482,21 @@ exports.render = function (req, res) {
   };
   let body = React.renderToString(frame);
   let json = JSON.stringify(clientPayload);
-  body = addReactScripts(body, `
+  let result = addReactScripts(
+    `<html>
+  ` + headString + `
```
Comment (ianb) [inline server/src/views/frame.js]: Shouldn't this just use `${headString}` ?

File: server/src/views/frame.js [hunk 2]
```diff
@@ -454,8 +511,13 @@ exports.renderSimple = function (req, res) {
...
     simple: true,
     shotDomain: req.url // FIXME: should be a property of the shot
-  });
+  };
+  let headString = React.renderToStaticMarkup(HeadFactory(serverPayload));
+  let frame = FrameFactory(serverPayload);
   let body = React.renderToStaticMarkup(frame);
-  body = '<!DOCTYPE HTML>\n' + body;
+  body = `<!DOCTYPE HTML>
+  ` + headString + `
```
Comment (ianb) [inline server/src/views/frame.js]: here too

**Comment by ianb** [comment]
Minor nit on the two string interpolations, otherwise good to go

File: server/src/views/frame.js (hunk 1 above)
Comment (fzzzy) [inline server/src/views/frame.js]: Hah. Of course. Doh.

**Pull request merged**